compassion_norway_payment/models/account_statement_import_netsgiro_parser.py

```diff
@@ -41,45 +40,41 @@ def parse(self, data_file, filename):
             path.basename(filename),
         )
         file_data = netsgiro.parse(StringIO(data_file.decode("iso-8859-1")).read())
-        lines = file_data.assignments[0].transactions
-        if not lines:
-            return currency_code, account_number, [{"name": name, "transactions": []}]
-        date = file_data.assignments[0].date
-        transactions = list(
-            itertools.chain.from_iterable(
-                map(lambda line: self._convert_line_to_transactions(line), lines)
+        assignement_dict_list = []
+        for assignement in file_data.assignments:
+            lines = assignement.transactions
+            if not lines:
+                assignement_dict_list.append({"name": name, "transactions": []})
+                continue
+            date = assignement.date
+            transactions = list(
+                itertools.chain.from_iterable(
+                    map(lambda line: self._convert_line_to_transactions(line), lines)
+                )
             )
-        )
-
+            assignement_dict_list.append(
+                    {
+                        "name": name,
+                        "date": date,
+                        "balance_start": float(-assignement.get_total_amount()),
+                        "balance_end_real": float(0),
+                        "transactions": transactions,
+                    })
         return (
             currency_code,
             account_number,
-            [
-                {
-                    "name": name,
-                    "date": date,
-                    "balance_start": float(-file_data.assignments[0].get_total_amount()),
-                    "balance_end_real": float(0),
-                    "transactions": transactions,
-                }
-            ],
+            [assignement_dict for assignement_dict in assignement_dict_list],
```

Comment:
Is there a reason to copy the list?
```suggestion
            assignement_dict_list,
```